ha/core/config/consul_kv_store.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
```

Comment:
Create a util directory in ha and put this module in that.

Reply:
Done